com.microsoft.mrtk.input/Simulation/Devices/SimulatedController.cs

```diff
@@ -308,17 +303,18 @@ public void ToggleNeutralPose()
                     const float jitterSeverity = 0.002f;
                     Vector3 jitter = jitterStrength * jitterSeverity * UnityEngine.Random.insideUnitSphere;
 
-                    SimulatedInputPosition += smoothedMoveDelta + jitter;
+                    CameraRelativePose = new Pose(
```

Comment:
Stylewise nit can we update each field of the camera relative pose rather than create a new one with new parameters

`CameraRelativePose.position += smoothedMoveDelta + jitter`

```
// If we not have been told to face the camera, apply the rotation delta.
if (rotationMode == ControllerRotationMode.UserControl)
{
       CameraRelativePose.rotation *= rotationDelta;
}
```

Reply:
Unfortunately nope, structs are immutable. Same reason you can't do `Vector3.x += 5`

Reply:
Can we do `CameraRelativePose.position = CameraRelativePose.position + smoothedMoveDelta + jitter` instead? I think that the property should still allow for setting a value. The issue with why `transform.position.x += 1;` doesn't work is because position is a property that returns a struct which doesn't allow the field value of x to be passed back down into the struct stored within transform.